habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -386,20 +386,20 @@ class EpisodicCompassSensor(HeadingSensor):
     def _get_uuid(self, *args: Any, **kwargs: Any) -> str:
         return self.cls_uuid
 
-    def get_agent_start_pos(self, episode, task):
-        return episode.start_position, quaternion_from_coeff(
+    def get_agent_start_rot(self, episode, task):
```

Comment:
Can we make rot -> rotation and pos -> position. Otherwise, there will be similar issues to what we saw here, in future, when someone else tries to use this class?

Reply:
also made pose->position_and_rotation